spec/controllers/miq_task_controller_spec.rb

```diff
@@ -264,12 +264,12 @@
                  :state_choice => "Initialized",
                  :time_period  => 6)
 
-        query = "userid=? AND ("\
-                "(state=? OR state=?) OR "\
-                "(state!=? AND state!=? AND state!=?)) AND "\
-                "updated_on>=? AND "\
-                "updated_on<=? AND "\
-                "state=?"
+        query = "jobs.guid IS NULL AND miq_tasks.userid=? AND ("\
+                "(miq_tasks.state=? OR miq_tasks.state=?) OR "\
+                "(miq_tasks.state!=? AND miq_tasks.state!=? AND miq_tasks.state!=?)) AND "\
+                "miq_tasks.updated_on>=? AND "\
+                "miq_tasks.updated_on<=? AND "\
+                "miq_tasks.state=?"
```

Comment:
I know you didn't write this.

but `miq_tasks.state` is confusing here (in these specs in general)